src/libs/SidebarUtils.js

```diff
@@ -251,10 +262,11 @@ function getOptionData(reportID) {
     }
 
     // If the last actor's details are not currently saved in Onyx Collection,
-    // then try to get that from the last report action.
+    // then try to get that from the last report action if that action is valid
+    // to get data from.
     let lastActorDetails = personalDetails[report.lastActorEmail] || null;
-    if (!lastActorDetails && lastReportActions[report.reportID] && lastReportActions[report.reportID].actionName !== CONST.REPORT.ACTIONS.TYPE.CREATED) {
-        const lastActorDisplayName = lodashGet(lastReportActions[report.reportID], 'person[0].text');
+    if (!lastActorDetails && visibleReportActionItems[result.reportID]) {
```

Comment:
NAB: They should be same but I think it makes sense to use report here cc @BeeMargarida 
```suggestion
    if (!lastActorDetails && visibleReportActionItems[report.reportID]) {
```